src/components/Charts/ChartVaccinations.tsx

```diff
@@ -1,37 +1,245 @@
-import React from 'react';
-import ChartZones from './ChartZones';
-import { VACCINATIONS_LEVEL_INFO_MAP } from 'common/metrics/vaccinations';
+import React, { useCallback, Fragment } from 'react';
+import moment from 'moment';
+import { isNumber } from 'lodash';
+import { AxisLeft, AxisBottom } from '@vx/axis';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleUtc, scaleLinear } from '@vx/scale';
+import { useTooltip } from '@vx/tooltip';
+import { formatPercent, formatUtcDate } from 'common/utils';
 import { Column } from 'common/models/Projection';
-import { formatPercent } from 'common/utils';
-import { getMetricDefinition, Metric } from 'common/metric';
+import { Tooltip, RectClipGroup } from 'components/Charts';
+import { ScreenshotReady } from 'components/Screenshot';
+import GridLines from 'components/Explore/GridLines';
+import * as Styles from 'components/Explore/Explore.style';
+import { Series } from 'components/Explore/interfaces';
+import ChartSeries, { SeriesMarker } from 'components/Explore/SeriesChart';
+import ChartOverlay from 'components/Explore/ChartOverlay';
+import { findPointByDate, getTimeAxisTicks } from 'components/Explore/utils';
+import * as ChartStyle from './Charts.style';
 
-const CAP_Y = 1;
+const getDate = (d: Column) => new Date(d.x);
+const getY = (d: Column) => d.y;
 
-const getPointText = (valueY: number) => formatPercent(valueY, 1);
+const xTickFormat = (date: Date, isMobile: boolean) => {
+  const momentDate = moment(date);
+  // Shows the year if the tick is in January (0) or December (11)
+  const dateFormat =
+    momentDate.month() === 0 || momentDate.month() === 11 ? 'MMM YYYY' : 'MMM';
+  return momentDate.format(dateFormat);
+};
+
+const VaccinesTooltip: React.FC<{
+  date: Date;
+  seriesList: Series[];
+  left: (d: Column) => number;
+  top: (d: Column) => number;
+  subtext: string;
+}> = ({ seriesList, left, top, date }) => {
+  const [seriesInitiated, seriesCompleted] = seriesList;
+  const pointCompleted = findPointByDate(seriesCompleted.data, date);
+  const pointInitiated = findPointByDate(seriesInitiated.data, date);
+
+  return pointCompleted && pointInitiated ? (
+    <Tooltip
+      width={'160px'}
```

Comment:
Is this the right place for this constant? (don't necessarily have anything better in mind, just flagging)

Reply:
I think so, other tooltips need more space. Each chart can have customized tooltips, and those might need more or less space, so it makes sense to pass the width as parameter